### PR TITLE
Add a comment

### PR DESCRIPTION
**Pull request by limavfabio**
This PR adds the functionality to add comments to each show, with the following features:
- Can't add empty comments
- Inputs alerts when a user tries to add empty comments
- show comments added in real-time
- Comments posted and fetched from InvolvementAPI

### CODE REVIEW

File: src/modules/commentsModal.js [hunk 1]
```diff
@@ -122,7 +128,41 @@ const generateModal = (id) => {
...
+              );
+              li.appendChild(liText);
+              ulComments.appendChild(li);
+            });
+          });
+        }
+      });
     }
   });
 };
```
Comment (LuisDavidRodriguez) [inline src/modules/commentsModal.js:168]: # Everything looks awesome my friend
- I'm just looking that here you have a lot of nested code inside a if statements, you can avoid that using return in each statement

File: src/modules/commentsModal.js [hunk 2]
```diff
@@ -122,7 +128,41 @@ const generateModal = (id) => {
...
+
+  // Event listener for the submit button
+  submitBtn.addEventListener('click', () => {
+    if (nameInput.value === '' || commentTextArea.value === '') {
+      nameInput.classList.add('empty-input');
+      commentTextArea.classList.add('empty-input');
+      setTimeout(() => {
+        nameInput.classList.remove('empty-input');
+        commentTextArea.classList.remove('empty-input');
+      }, 1000);
```
Comment (LuisDavidRodriguez) [inline src/modules/commentsModal.js:142]: for example this statement you can finalise it just like this
``` 
if (nameInput.value === '' || commentTextArea.value === '') {
      nameInput.classList.add('empty-input');
      commentTextArea.classList.add('empty-input');
      setTimeout(() => {
        nameInput.classList.remove('empty-input');
        commentTextArea.classList.remove('empty-input');
      }, 1000);
     

            return;  // <-- this is the key
```

so as soon as your code has something empty you, stop the execution of the function up to that point and in that way you can avoid the `else` statement that enclose everything that will happen if you don't have empty values

File: src/modules/commentsModal.js (hunk 1 above)
Comment (limavfabio) [reply src/modules/commentsModal.js:168]: Thank you for your time! I will take a look into that.

File: src/modules/commentsModal.js (hunk 2 above)
Comment (limavfabio) [reply src/modules/commentsModal.js:142]: Thank you for the heads up! I'll take a look into that.

**Pull request merged**